setup.py

```diff
@@ -11,7 +11,9 @@
     "pytest-cov",
     "pytest-docker-fixtures[pg]>=1.3.0",
     "prometheus-client>=0.9.0",  # TODO: remove
-    "docker>=6.0.0,<=6.1.3"
+    "docker>=6.0.0,<=6.1.3",
+    # Until this is merged https://github.com/docker/docker-py/pull/3257
+    "requests==2.31.0"
```

Comment:
Do you think `requests<=2.31.0` would work as well? I think it would be better to be less restrictive?

Reply:
sure, I just deleted that, docker 7.1.0 was released some hours ago and solve this issue, I have pinned docker to this version wuth docker>=7.1.0